### PR TITLE
#178 Kafka 0.10.2.1

### LINKED ISSUE #178: Kafka 0.10.2.0
This is created to track our upgrade to support Kafka 0.10.2.0 client.

### PR DESCRIPTION
**Pull request by original-brownbear**
fixes #178 

* Upgraded Kafka to `0.10.2.1`
* Bump version
* Gitignore build files

### CODE REVIEW

**Comment by suyograo** [comment]
waiting for CI

**Comment by original-brownbear** [comment]
@suyograo output PR here https://github.com/logstash-plugins/logstash-output-kafka/pull/133

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## 6.2.8
```
Comment (suyograo) [inline CHANGELOG.md]: Can we bump this to 6.3.0 since we're bringing a big(ish) change from Kafka?

**Comment by suyograo** [comment]
@original-brownbear change the version and merge.

**Pull request closed without merging**